pinax/likes/views.py

```diff
@@ -1,49 +1,48 @@
-import json
-
-from django.http import HttpResponse, HttpResponseForbidden
-from django.shortcuts import get_object_or_404, redirect
+from django.contrib.auth.decorators import login_required
```

Comment:
We should use DUA's `login_required` decorator if DUA is installed and fall back to contrib auth's version. However, since we are doing CBVs, we should use DUA's `account.mixins.LoginRequiredMixin` if available, otherwise fall back.